snippets/multisigning.py

```diff
@@ -0,0 +1,69 @@
+"""Example of how we can multisign a transaction"""
+from xrpl.clients import JsonRpcClient
+from xrpl.models.requests import SubmitMultisigned
+from xrpl.models.transactions import AccountSet, SignerEntry, SignerListSet
+from xrpl.transaction import (
+    autofill,
+    multisign,
+    safe_sign_and_autofill_transaction,
+    send_reliable_submission,
+)
+from xrpl.utils import str_to_hex
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234/")
+
+master_wallet = generate_faucet_wallet(client, debug=True)
+signer_wallet_1 = generate_faucet_wallet(client, debug=True)
+signer_wallet_2 = generate_faucet_wallet(client, debug=True)
+
+signer_entries = [
+    SignerEntry(account=signer_wallet_1.classic_address, signer_weight=1),
+    SignerEntry(account=signer_wallet_2.classic_address, signer_weight=1),
+]
+signer_list_set_tx = SignerListSet(
+    account=master_wallet.classic_address,
+    signer_quorum=2,
+    signer_entries=signer_entries,
+)
+signed_signer_list_set_tx = safe_sign_and_autofill_transaction(
+    signer_list_set_tx, master_wallet, client
+)
+signed_list_set_tx_response = send_reliable_submission(
```

Comment:
I'd recommend adding a print statement each time right before you submit in order to let the user know that the code is still working. (Otherwise it just looks like it's stuck)

Ex. (If you change this, you'll also want to change the print statement below)
```suggestion
print('Constructed SignerListSet and submitting it to the ledger...')
signed_list_set_tx_response = send_reliable_submission(
```